pkg/config/loader.go

```diff
@@ -0,0 +1,93 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/BurntSushi/toml"
+
+	"github.com/ipfs/testground/pkg/logging"
+)
+
+const (
+	EnvTestgroundHomeDir = "TESTGROUND_HOME"
+
+	DefaultListenAddr = "localhost:8042"
+)
+
+func (e *EnvConfig) Load() error {
```

Comment:
The `*EnvConfig` now has a `Load()` method that loads the configuration, applies any env variables, and fallbacks.